class/defaults.yml

```
@@ -2,3 +2,112 @@ parameters:
   topolvm:
     =_metadata: {}
     namespace: syn-topolvm
+
+    deviceclasses:
+      - name: ssd
+        volumegroup: vgssd
+        sparegb: 1
+
+    storageclasses:
+      - name: ssd-local
+        fstype: xfs
+        isdefault: false
+        volumeexpansion: true
+
+
+    # Resource limits
+    resources:
+      scheduler:
+        requests:
+          memory: 50Mi
+          cpu: 50m
+        limits:
+          memory: 200Mi
+          cpu: 200m
+      lvmd:
+        requests:
+          memory: 100Mi
+          cpu: 100m
+        limits:
+          memory: 500Mi
+          cpu: 500m
+      node:
+        requests:
+          memory: 100Mi
+          cpu: 100m
+        limits:
+          memory: 500Mi
+          cpu: 500m
+      controller:
+        requests:
+          memory: 50Mi
+          cpu: 50m
+        limits:
+          memory: 200Mi
+          cpu: 200m
+
+    helmValues:
+      image:
+        # image.repository -- TopoLVM image repository to use.
+        repository: quay.io/topolvm/topolvm-with-sidecar
+        # image.tag -- TopoLVM image tag to use.
+        tag: 0.8.3
```

Generally, we recommend providing container image repositories and tags in a top-level parameter `images`:

```
parameters:
  topolvm:
    images:
      topolvm:
        registry: quay.io
        repository: topolvm/topolvm-with-sidecar
        tag: 0.8.3
```

And using those parameters where the image is required:

```suggestion
      image:
        # image.repository -- TopoLVM image repository to use.
        repository: ${topolvm:images:topolvm:registry}/${topolvm:images:topolvm:repository}
        # image.tag -- TopoLVM image tag to use.
        tag: ${topolvm:images:topolvm:tag}
```